compose/cli/main.py

```diff
@@ -308,13 +308,24 @@ def logs(self, project, options):
         Usage: logs [options] [SERVICE...]
 
         Options:
-            --no-color  Produce monochrome output.
+            --no-color          Produce monochrome output.
+            -f, --follow        Follow log output.
+            -t, --timestamps    Show timestamps.
+            --tail="all"        Number of lines to show from the end of the logs
+                                for each container.
         """
         containers = project.containers(service_names=options['SERVICE'], stopped=True)
 
         monochrome = options['--no-color']
+        log_args = {}
+        log_args['stream'] = options['--follow']
+        log_args['timestamps'] = options['--timestamps']
+        tail = options['--tail']
+        if tail is not None and tail.isdigit():
+            tail = int(tail)
+        log_args['tail'] = tail
```

Comment:
Small problem here, we should show an error if `--tail` is invalid instead of just ignoring it.   

`image_type_from_opt()` is a good example for this. It lets you move this `if` branch into the validation function as well.